.changeset/tabaxi-fellas-inspire.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
None of the code examples use the methods on the table node, which I think are one of the main reasons this library exists and the main way it should be used.

A simple example creating a table, creating a table with three columns, two of the same type, and one of a different type, then inserting some rows with cell and reading and editing a cell using the types from the column would make for a nice simple example.

Since there are no examples of it yet, do we have an API for doing strongly types reading of cells using the column?

Also we might want to show allowing row and cell ids to default to uuids. I know that will cause issue using the methods currently on unhydrated nodes, but I'm working on fixes for that and it might be worth highlighting the problem.

Reply:
I'll add some more usage examples.